outreaches.md

```diff
@@ -0,0 +1,496 @@
+---
+
+# Outreach
+
+This document defines the Outreach resource. 
+
+Outreachs are a type of action that a user may take by participating in an [advocacy campaign](advocacy_campaigns.html). Outreaches have fields to describe them such as when the outreach was created, who the user contacted as the target of their advocacy, and the content of the message a user sent to that target, and typically are linked to the person who made the outreach.
+
+
+### Sections
+
+* [Endpoints and URL structures](#endpoints-and-url-structures)
+* [Fields](#fields)
+    * [Common Fields](#common-fields)
+    * [Outreach Fields](#outreach-fields)  
+    * [Related Objects](#related-objects)
+    * [Links](#links)
+* [Helpers](#helpers)
+* [Related Resources](#related-resources)
+* [Scenarios](#scenarios)
+    * [Scenario: Retrieving a collection of Outreach resources (GET)](#scenario-retrieving-a-collection-of-outreach-resources-get)
+    * [Scenario: Retrieving an individual Outreach resource (GET)](#scenario-scenario-retrieving-an-individual-outreach-resource-get)
+    * [Scenario: Creating a new outreach (POST)](#scenario-creating-a-new-outreach-post)
+    * [Scenario: Modifying an outreach (PUT)](#scenario-modifying-an-outreach-put)
+    * [Scenario: Deleting an outreach (DELETE)](#scenario-deleting-an-outreach-delete)
+
+
+{% include endpoints_and_url_structures.md %}
+
+The link relation label for a Outreach resource is ```osdi:outreach``` for a single Outreach resource or ```osdi:outreaches``` for a collection of Outreach resources.
+
+_[Back to top...](#)_
+
+
+## Fields
+
+{% include fields_intro.md %}
+
+{% include global_fields.md %}
+
+_[Back to top...](#)_
+
+
+### Outreach Fields
+
+A list of fields specific to the Outreach resource.
+
+| Name          | Type      | Description
+|-----------    |-----------|-----------|--------------
+|origin_system		|string     |A human readable identifier of the system where this outreach was created. (ex: "OSDI System")
+|action_date		|string		|The date and time the outreach was made by the person.
+|type				|flexunum	|The type of outreach, specifying how the user performed the outreach to targets. One of "email", "in-person", "phone", "postal mail", or another type as needed.
+|duration		|integer		|The duration in seconds of the outreach, if applicable. (ex: duration will only be present on phone outreach types)
+|subject		|string		|The subject of the outreach, if applicable. (ex: subject will only be present on email outreach types)
+|message		|string		|The message of the outreach, if applicable. (ex: message will only be present on email or postal mail outreach types)
+|targets			|[Target[]](#target)    |An array of target object hashes representing the targets of the outreach.
+
+_[Back to top...](#)_
+
+
+### Related Objects
+
+These JSON hashes included in the table above are broken out into their own tables for readability, rather than independent resources with their own endpoints.
+
+#### Target

```

Comment:
NB that Target is also what we call "the person you attempted to contact during a Canvass" in PR 199.  They are of course very similar conceptually, although very different in terms of the data model.  So that might or might not be ok, just a thought.